### DISCUSSION TITLE
Can anybody help to get github repo of this documentation section

### DISCUSSION THREAD

**Discussion by JayeshYadav99** [appwrite/appwrite#5575] [General]
Can anybody help to get github repo of this partcular  documentation section [https://appwrite.io/docs/authentication](url),i want to contribute to this docs ,but i cannot find this section in appwrite docs folder

**Comment by joeyouss** [comment]
Unfortunately, you won't be able to contribute to this doc since they are generated from route descriptions https://github.com/appwrite/appwrite/blob/181741d7a2082f5c9d3ca727384601f3cf75e5bf/app/controllers/api/account.php#L3 . 

We're working on making doc contributions easier, at the moment it's not possible since we have some private repos as well. That being said, we have many other issues you can contribute to!

**Comment by JayeshYadav99** [comment]
Thanks For Update
